### PR TITLE
Wording revision across entire app

### PR DESCRIPTION
**Pull request by dasilvarosa**
Thorough improvement of in-app wording. 

### Type

UI Improvement

### Context

1. User testing and feedback indicated texts need to be further simplified.
2. Simplified wordings have been aligned with mobile app and reduce word counts for translation (e.g. French produces more/longer words).

### Parts of the app affected / Test plan

Some changes might need some minor development, notably formatting in the on-boarding and potentially 2 icons for PIN entry on Ledger Nano S.

### CODE REVIEW

File: static/i18n/en/app.json [hunk 1]
```diff
@@ -530,34 +530,34 @@
...
       "initialize": {
         "title": "Choose your PIN code",
         "instructions": {
           "nano": {
-            "step1": "Connect the Ledger Nano S to your computer.",
-            "step2": "Press both buttons simultaneously as instructed on the screen.",
-            "step3": "Press the right button to select <1><0>Configure as new device?</0></1>",
-            "step4": "Choose a PIN code between 4 and 8 digits long, followed by the checkmark (✓)."
+            "step1": "Connect the Ledger Nano S to your computer. Press both buttons to begin.",
+            "step2": "Press the right button to select *Configure as new device?*.",
```
Comment (gre) [inline static/i18n/en/app.json]: please move to `<1><0>` and one of our dev need to refactor some code

File: static/i18n/en/app.json [hunk 2]
```diff
@@ -530,34 +530,34 @@
...
+        "note2": "Use 8 digits for optimal security.",
+        "note3": "Never use a device supplied with a PIN code or a recovery phrase."
       },
       "initialize": {
         "title": "Choose your PIN code",
         "instructions": {
           "nano": {
-            "step1": "Connect the Ledger Nano S to your computer.",
-            "step2": "Press both buttons simultaneously as instructed on the screen.",
-            "step3": "Press the right button to select <1><0>Configure as new device?</0></1>",
```
Comment (gre) [inline static/i18n/en/app.json]: important for the dev: the `<1><0>` of step3 is gone, now in step2

File: static/i18n/en/app.json [hunk 3]
```diff
@@ -530,34 +530,34 @@
...
       "restore": {
         "title": "Choose your PIN code",
         "instructions": {
           "nano": {
-            "step1": "Connect the Ledger Nano S to your computer.",
-            "step2": "Press both buttons simultaneously as instructed on the screen.",
-            "step3": "Press the left button to cancel <1><0>Initialize as new device?</0></1>. Then press the right button to select <3><0>Restore configuration?</0></3>",
-            "step4": "Choose a PIN code between 4 and 8 digits long, followed by the checkmark (✓)."
+            "step1": "Connect the Ledger Nano S to your computer. Press both buttons to begin.",
+            "step2": "Press the left button to cancel *Initialize as new device?*. Then press the right button to select *Restore configuration?*.",
```
Comment (gre) [inline static/i18n/en/app.json]: same

File: static/i18n/en/app.json [hunk 4]
```diff
@@ -570,44 +570,44 @@
     "writeSeed": {
       "initialize": {
         "title": "Save your recovery phrase",
-        "desc": "Your device will generate a 24-word recovery phrase to back up your private keys",
+        "desc": "Your device will display a 24-word recovery phrase to back up your private keys",
         "nano": {
-          "step1": "Copy the word displayed below <1><0>Word #1</0></1> in position 1 on a blank Recovery sheet.",
-          "step2": "Press the right button to display <1><0>Word #2</0></1> and repeat the process until all 24 words are copied on the Recovery sheet.",
```
Comment (gre) [inline static/i18n/en/app.json]: one dev need to acknowledge this as well.

**Comment by gre** [comment]
@Arnaud97234 can you test this?

**Comment by Arnaud97234** [comment]
@dasilvarosa. Can you give us a feedback on my comment above please.

**Comment by dasilvarosa** [comment]
@Arnaud97234 PIN code texts fixed. The technical data modal should be reduced to 3 bullet points. @juan-cortes could you do that please?

**Comment by juan-cortes** [comment]
@dasilvarosa I created a pr on your branch, there were extra items on both analytics and technical data modals.

**Comment by dasilvarosa** [comment]
Thanks @juan-cortes. @Arnaud97234 can you review?

**Pull request merged**